modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -0,0 +1,82 @@
+= Updating the Runtime Version for CloudHub Apps 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, runtime version, mule, updates,
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+To provide a safe cloud environment for applications and data, MuleSoft regularly scans apps deployed on CloudHub and identifies apps that require updates to the CloudHub runtimes.
+These updates provide the latest OS and security patches.
+To keep your apps up-to-date and secure, apply the updates when they become available.
+
+Because these updates contain required security patches, you cannot roll back these updates once applied.
+
+
+You can schedule and update your apps manually at your convenience or you can wait for MuleSoft to apply the updates automatically.
+
+MuleSoft updates CloudHub apps automatically, based on the environment, on this schedule:
+
+[%header,cols="2"]
+|===
+|Environment |App Update Schedule 
+|*Production*|Every three months
+|*Sandbox*|Once every month
+|===
+
+When a new version is available and CloudHub application updates are required, the *Applications* page displays a banner and the *Update Required* tab lists the apps that require updates:
+
```

Comment:
Can the section above this be done away with for now and replaced with something like this:

"When you run an application on CloudHub, you get access to validated OS updates, security patches and bug fixes on a regular basis. You can keep you applications on CloudHub applications upto date by manually take the patch updates at your convenience or let the platform update these applications for you regularly.

When a new date patch version of the mule runtime your application is running is available the
Applications page displays a banner and the Update Required tab lists the apps that require
updates:"

We will revisit this once the strategy is crystalized.